installation/designer.md

```diff
@@ -4,52 +4,46 @@ layout: documentation
 
 {% include base.html %}
 
-# Installing the designer
+# Eclipse SmartHome Designer
 
-The Designer is an eclipse-based IDE allowing to edit text configuration file.
+The Eclipse SmartHome Designer is an eclipse-based IDE specialized for Eclipse SmartHome and openHAB configuration file management.
+While your openHAB setup may run on a dedicated system, like a [Raspberry Pi](rasppi.html), the Designer can be installed and executed on your personal PC or Mac. In order to configure your openHAB setup, you will need a mounted network share to your configuration. 
 
-## Installation
+## Setup
 
-* the latest version is available for download here: [Eclipse SmartHome Designer Downloads](https://github.com/eclipse/smarthome/blob/master/docs/documentation/community/downloads.md#designer-builds)
-* get the designer for your plattform (Windows/OSX/Linux)
-* as the SmartHome Designer is still in development, consider selecting the nightly snapshots build
+* The latest version is available for download at: [Eclipse SmartHome Designer Downloads](https://github.com/eclipse/smarthome/blob/master/docs/documentation/community/downloads.md#designer-builds)
+* Get the designer for your plattform (Windows/OSX/Linux)
+* As the SmartHome Designer is still in development, consider selecting the nightly snapshots build
 
 ![](images/smarthone-designer-05.png)
 
-* navigate to the `openHAB` folder we created during the openHAB installation
-* create a new folder called `extra`
-* move the downloaded file (e.g. `eclipsesmarthome-incubation-0.9.0-SNAPSHOT-designer.zip`) inside the created folder
+The downloaded `.zip` archive contains the Designer executables. 
+* **Windows**: Extract the archive to a path of your choosing, e.g. `C:\EclipseSmarthomeDesigner`
+* **Linux**: Extract the archive to a path of your choosing, e.g. `/opt/eclipsesmarthomedesigner`
+* **Max OSX**: move the archive to your programs
```

Comment:
I have no personal experience with Mac OSX. Can you please update this one line appropriately. thanks ;)